contracts/utils/RelayRecipient.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.6.4;
+pragma solidity ^0.6.12;
```

Comment:
We should explicitly set the version for all non-interface/util contracts (remove the caret)